lib/__tests__.js

```diff
@@ -2,7 +2,7 @@
 
 const Test = require('tape');
 const gql = require('graphql-tag');
-const { SchemaDirectiveVisitor } = require('@graphql-tools/utils');
+const { SchemaDirectiveVisitor } = require('graphql-tools');
```

Comment:
i could be wrong but don't they recommend using the more specific graphql-tools packages instead of the top level `graphql-tools` package?

Reply:
Yes but I didn't want to do the full migration yet. We've historically used this older package and a lot of breaking changes in newer packages (like schema directive visitor stuff).